triage/templates/triage/wizard-step-summary.html

```diff
@@ -29,9 +33,5 @@ <h3 class="title">You told us:</h3>
 {% endblock %}
 
 {% block submit_button_label %}
-	{% if is_user_reviewing_persisted_answers %}
-		Continue my exporting journey
-	{% else %}
-		Create my exporting journey
-	{% endif %}
```

Comment:
these carriage returns were causing odd UI: the button renders the carriage returns so the button is large.